.github/workflows/lint.yml

```diff
@@ -150,22 +150,25 @@ jobs:
         env:
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
-          mkdir flake8-output
+          mkdir -p flake8-output
           cd flake8-output
           echo "$HEAD_SHA" > commit-sha.txt
-      - name: Run flake8
+      - name: Install dependencies
         run: |
           set -eux
           pip install typing-extensions # for tools/translate_annotations.py
           pip install -r requirements-flake8.txt
           flake8 --version
+      - name: Run flake8
+        run: |
+          set -eux
           flake8 | tee "${GITHUB_WORKSPACE}"/flake8-output.txt
-          cp flake8-output.txt flake8-output/annotations.json
       - name: Translate annotations
         if: github.event_name == 'pull_request'
         env:
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
+          cp "${GITHUB_WORKSPACE}"/flake8-output.txt flake8-output/annotations.json
```

Comment:
I added this line in the first place but I'm gonna be honest, I have no idea what it's supposed to do 😅 could you remove it please? It doesn't serve any purpose especially if it's being run in the "Translate annotations" step (before at least it wasn't gated by `github.event_name == 'pull_request'` so maybe it had some purpose, but I think it should be removed anyways).